<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_record</class>
 <widget class="QDialog" name="add_record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>691</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>增加记录行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reduce">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>减少记录行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_ok">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_exit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="showtime_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
